radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/RadixParticleStore.java

```diff
@@ -18,7 +23,14 @@ public RadixParticleStore(AtomStore atomStore) {
 		return cache.computeIfAbsent(address, addr -> atomStore.getAtoms(address)
 			.flatMap(observation -> Observable.create(emitter -> {
 				if (observation.hasAtom()) {
-					observation.getAtom().spunParticles().filter(s -> s.getParticle().getShardables().contains(address))
+					final List<SpunParticle> spunParticleList = observation.getAtom().spunParticles().collect(Collectors.toList());
+
+					if (observation.getType() == Type.DELETE) {
```

Comment:
Is this only because Core doesn't return DELETE Atoms in the right order? 
If so, we should make a task for that

Reply:
No, it's because currently particle reducers can only handle particle DELETEs if done in reverse order due to the way it manages state.